.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,48 @@
+name: Chromatic
+
+on: [push]
+
+jobs:
+  chromatic:
+    timeout-minutes: 20
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        project:
+          [designer-app, css-engine, design-system, icons, image, react-sdk]
+        include:
+          - project: designer-app
+            chromaticToken: ${{ secrets.CHROMATIC_DESIGNER_APP_TOKEN }}
```

Comment:
Lol, GitHub is complaining about this line... it seems that perhaps secrets cant be used in the matrix. I need to investigate this.

Reply:
Ok, I think that I've solved this here: https://github.com/webstudio-is/webstudio-designer/pull/519/commits/8c5ad32c74e9b954f13feccafe796cd66508d949